src/ops/shell.ts

```diff
@@ -21,7 +21,8 @@ const shell = async (
         debug('result %o', res)
       } catch (error) {
         if (sh_ignore_exit !== true) {
-          logger.err(error.stderr)
+          logger.err(error)
```

Comment:
`error.stderr` was undefined.
The new debug line will help you get more details.